cf-net/cf-net.h

```diff
@@ -0,0 +1,73 @@
+/*
+   Copyright (C) CFEngine AS
+
+   This file is part of CFEngine 3 - written and maintained by CFEngine AS.
+
+   This program is free software; you can redistribute it and/or modify it
+   under the terms of the GNU General Public License as published by the
+   Free Software Foundation; version 3.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
+
+  To the extent this program is licensed as part of the Enterprise
+  versions of CFEngine, the applicable Commercial Open Source License
+  (COSL) may apply to this file if you as a licensee so wish it. See
+  included file COSL.txt.
+*/
+
+#ifndef CFENGINE_CF_NET_H
+#define CFENGINE_CF_NET_H
+
+#include <platform.h>
+#include <client_code.h>        // AgentConnection
+
+#define CF_NET_VERSION "0.1.0"
+
+typedef struct
```

Comment:
What is the reason of this file `cf-net.h`? How many of these exported functions are really needed to be exported? The rest you should make them `static` and remove them from the header file.